frontend/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "aghtochess",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@types/jest": "^24.0.0",
```

Comment:
types should be `devDependencies` should they not?

Reply:
Not really, this might matter in a node.js app, here we only care about the final generated bundle, which does not contain type information either way.

Reply:
Ok.